test/parallel/test-event-emitter-remove-listeners.js

```diff
@@ -1,104 +1,106 @@
-
-var e2 = new events.EventEmitter();
-e2.on('hello', listener1);
-e2.on('removeListener', common.fail);
-e2.removeListener('hello', listener2);
-assert.deepStrictEqual([listener1], e2.listeners('hello'));
-
-var e3 = new events.EventEmitter();
-e3.on('hello', listener1);
-e3.on('hello', listener2);
-e3.once('removeListener', common.mustCall(function(name, cb) {
-  assert.equal(name, 'hello');
-  assert.equal(cb, listener1);
-  assert.deepStrictEqual([listener2], e3.listeners('hello'));
-}));
-e3.removeListener('hello', listener1);
-assert.deepStrictEqual([listener2], e3.listeners('hello'));
-e3.once('removeListener', common.mustCall(function(name, cb) {
-  assert.equal(name, 'hello');
-  assert.equal(cb, listener2);
-  assert.deepStrictEqual([], e3.listeners('hello'));
-}));
-e3.removeListener('hello', listener2);
-assert.deepStrictEqual([], e3.listeners('hello'));
-
-var e4 = new events.EventEmitter();
-e4.on('removeListener', common.mustCall(function(name, cb) {
-  if (cb !== remove1) return;
-  this.removeListener('quux', remove2);
-  this.emit('quux');
-}, 2));
-e4.on('quux', remove1);
-e4.on('quux', remove2);
-e4.removeListener('quux', remove1);
-
-var e5 = new events.EventEmitter();
-e5.on('hello', listener1);
-e5.on('hello', listener2);
-e5.once('removeListener', common.mustCall(function(name, cb) {
-  assert.equal(name, 'hello');
-  assert.equal(cb, listener1);
-  assert.deepStrictEqual([listener2], e5.listeners('hello'));
-  e5.once('removeListener', common.mustCall(function(name, cb) {
-    assert.equal(name, 'hello');
-    assert.equal(cb, listener2);
-    assert.deepStrictEqual([], e5.listeners('hello'));
+{
+  const ee = new events.EventEmitter();
+  ee.on('hello', listener1);
+  ee.on('hello', listener2);
+  ee.once('removeListener', common.mustCall(function(name, cb) {
+    assert.strictEqual(name, 'hello');
+    assert.strictEqual(cb, listener1);
+    assert.deepStrictEqual([listener2], ee.listeners('hello'));
+    ee.once('removeListener', common.mustCall(function(name, cb) {
+      assert.strictEqual(name, 'hello');
+      assert.strictEqual(cb, listener2);
+      assert.deepStrictEqual([], ee.listeners('hello'));
+    }));
+    ee.removeListener('hello', listener2);
+    assert.deepStrictEqual([], ee.listeners('hello'));
   }));
-  e5.removeListener('hello', listener2);
-  assert.deepStrictEqual([], e5.listeners('hello'));
-}));
-e5.removeListener('hello', listener1);
-assert.deepStrictEqual([], e5.listeners('hello'));
-
-const e6 = new events.EventEmitter();
-
-const listener3 = common.mustCall(() => {
-  e6.removeListener('hello', listener4);
-}, 2);
+  ee.removeListener('hello', listener1);
+  assert.deepStrictEqual([], ee.listeners('hello'));
+}
 
-const listener4 = common.mustCall(() => {}, 1);
+{
+  const ee = new events.EventEmitter();
+  const listener3 = common.mustCall(() => {
+    ee.removeListener('hello', listener4);
+  }, 2);
+  const listener4 = common.mustCall(() => {}, 1);
```

Comment:
Would it be worth to use arrow functions in the rest of the code?
The `,1` is redundant and could be removed while we're here.